PowerEditor/src/Parameters.cpp

```diff
@@ -839,6 +839,9 @@ winVer NppParameters::getWindowsVersion()
    {
 		case VER_PLATFORM_WIN32_NT:
 		{
+			if (osvi.dwMajorVersion == 10 && osvi.dwMinorVersion == 22000)

```

Comment:
@donho @LeonC510 
this line is wrong
it should be `if (osvi.dwMajorVersion == 10 && osvi.dwMinorVersion == 0 && osvi.dwBuildNumber >= 22000)`

Reply:
@ozone10 
It works. Thank you for the tip!